docs/integrations/spark.md

```diff
@@ -191,157 +185,128 @@ df.write.partitionBy("example-column").parquet(s"s3a://${repo}/${branch}/output-
 
 The data is now created in lakeFS as new changes in your branch. You can now commit these changes or revert them.
 
-## Access lakeFS using the lakeFS-specific Hadoop FileSystem
+## Use the lakeFS-specific Hadoop FileSystem
 
-To use this mode, you configure the Spark application to perform metadata operations on the
-lakeFS server and all data operations directly through the same underlying object store that
-lakeFS uses. The lakeFS FileSystem currently supports Spark with Hadoop Apache 2.7 using only the S3A Hadoop FileSystem
```

Comment:
> using only the S3A Hadoop FileSystem

I think we should keep this limitation and make it clear that lakefsfs only works with s3a

Reply:
Added a clarification that this mode is only relevant for S3 users.